### DISCUSSION TITLE
Repo landing page README not updating

### DISCUSSION THREAD

**Discussion by jessicarowell** [community/community#102649] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

We've updated the main README.md file in our repo, but the repo's landing page still shows the outdated one.  If you click on the actual README.md file **from dev branch**, it's the updated (much shorter) one.  Anyone have any idea what's going on?

Repo: [Tostadas](https://github.com/CDCgov/tostadas/tree/dev)

**Comment by DJLink** [comment]
Your main branch seems to be master, but the README.md changes were made on dev branch, dev branch appears to be showing the correct README.md correct?
So in order for your landing page to show the README.md contents from dev, you have to merge the dev branch onto master. Or alternatively change your main branch from master to dev (not the ideal solution probably)

**Comment by Gecko51** [comment]
Yep, DJLink nailed it. The repo landing page always renders the README from the default branch, not from whichever branch happens to have the newest commits.

So if your default branch is `master` and you've only committed the new README to `dev`, the landing page will keep showing the old content until one of these happens:

- You merge `dev` into `master` (usual flow via a PR).
- You change the default branch to `dev` in `Settings` > `Branches` > `Default branch`.

A few extra things that sometimes cause the same symptom:

- Browser cache. After pushing to the default branch, hit the landing page with a hard reload (cmd/ctrl + shift + R) to make sure you're not looking at a cached version.
- README location. GitHub picks the first README it finds in this order: `.github/README.md` → `README.md` (root) → `docs/README.md`. If you've got more than one, the higher-priority one wins and the others get ignored on the landing page.
- Profile READMEs (repo with the same name as the user/org) follow the same default-branch rule.

Easy way to confirm which version GitHub considers canonical: go to `https://github.com/<owner>/<repo>/blob/<default-branch>/README.md` directly. Whatever shows there is what the landing page will render.

Once the merge lands, the landing page updates within a few seconds, no manual refresh needed on GitHub's side.
